.github/workflows/build.yml

```diff
@@ -66,18 +66,19 @@ jobs:
             use_sccache: false
             use_llvm: false
             build_wasm: false
+            # [todo] xdoardo: Reinstate when the code we generate for aarch64 is working correctly. 
+            # llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/18.x/llvm-darwin-aarch64.tar.xz'
           - build: windows-x64
             os: windows-2019
             artifact_name: 'wasmer-windows-amd64'
-            llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/15.x/llvm-windows-amd64.tar.xz'
+            llvm_url: 'https://github.com/xdoardo/llvm-custom-builds/releases/download/18.x/llvm-windows-amd64.tar.xz'
```

Comment:
Fixme prior to merging

Reply:
[303eda9](https://github.com/wasmerio/wasmer/pull/5086/commits/303eda9e57eff603d0eb08002be15a642ad72ed5)